site/content/3.12/release-notes/version-3.12/whats-new-in-3-12.md

```diff
@@ -222,5 +222,59 @@ of outgrowing the maximum number of file descriptors the ArangoDB process
 can open. Thus, these options should only be enabled on deployments with a
 limited number of collections/shards/indexes.
 
+## Client tools
+
+### arangodump
+
+#### Improved dump performance
+
+ArangoDB 3.12 includes extended parallelization capabilities to work not only
```

Comment:
At the moment, it is tagged as an experimental feature, which I think we should mention here:
https://github.com/arangodb/arangodb/blob/4f154796969214eca5c25e143afb87fb9a16060d/client-tools/Dump/DumpFeature.cpp#L914

Reply:
The "experimental" tagging will be removed by this upstream PR:
https://github.com/arangodb/arangodb/pull/20022